packages/client/components/InsightsDomainPanel.tsx

```diff
@@ -147,10 +198,24 @@ const InsightsDomainPanel = (props: Props) => {
     activeOrganizationCount,
     activeTeamCount,
     activeUserCount,
-    meetingCount
+    meetingCount,
+    organizations
   } = domain
-
+  const toBeLockedOrg = organizations.find((org) => org.scheduledLockAt)
   const isDesktop = useBreakpoint(Breakpoint.NEW_MEETING_SELECTOR)
+  const atmosphere = useAtmosphere()
+  const {history} = useRouter()
+
+  const handleClick = () => {
+    if (!toBeLockedOrg) return
+    const {id: orgId} = toBeLockedOrg
+    SendClientSegmentEventMutation(atmosphere, 'Upgrade CTA Clicked', {
+      upgradeCTALocation: 'usageStats',
```

Comment:
@tghanken just checking if this is how you'd like to record the event when a user clicks the "Upgrade" button?

Reply:
For tracking Meeting History (#7511), we started using `Upgrade Intent` rather than `Upgrade CTA Clicked` to track across more areas.